src/patterns/task-list-pages/index.md

```diff
@@ -18,7 +16,7 @@ Task list pages help users understand:
 - the order they should complete tasks in
 - when they've completed tasks
 
-![A screenshot showing an example of the task list page, includes a heading, and three grouped sections that each contain tasks, some of these tasks are marked as completed.](task-list-whole.svg)
+{{ example({group: "patterns", item: "task-list-pages", example: "default", html: true, nunjucks: true, open: false}) }}
```

Comment:
Examples for other page patterns include a header and footer in the example – see the [service unavailable pattern](https://design-system.service.gov.uk/patterns/service-unavailable-pages/). It’s worth adding those to remain consistent.

Reply:
Good point. Will fix.

Reply:
@stevenjmesser Fixed in 82373e3b2ca54e43ca571e57bde18403e22d5781 thanks